### PR TITLE
17.2.6 Release notes

### PR DESCRIPTION
**Pull request by yuriw**

### CODE REVIEW

**Comment by yuriw** [comment]
to render see https://wip-yuriw-release-17.2.6-post-main.ceph.io/en/news/blog/2023/v17-2-6-quincy-released/

File: src/en/news/blog/2023/v17-2-6-quincy-released/index.md [hunk 1]
```diff
@@ -0,0 +1,1052 @@
...
+  mechanism has been added: `ceph-objectstore-tool` got `trim-pg-log-dups` op
+  that targets situations where OSD is unable to boot due to those inflated dups.
+  If that is the case, in OSD logs the "You can be hit by THE DUPS BUG" warning
+  will be visible.
+  Relevant tracker: https://tracker.ceph.com/issues/53729
+
+- RBD: The `rbd device unmap` command gained the `--namespace` option.  Support for
+  namespaces was added to RBD in Nautilus 14.2.0; and while it has been possible to
+  map and unmap images in namespaces using the `image-spec` syntax since then
+  but the corresponding option available in most other commands was missing.
```
Comment (ljflores) [inline src/en/news/blog/2023/v17-2-6-quincy-released/index.md:30]: Changelog looks good, but this "Notable changes" section looks different compared to what you have in https://github.com/ceph/ceph/pull/50721.

Comment (yuriw) [reply src/en/news/blog/2023/v17-2-6-quincy-released/index.md]: fixed, good catch, thx @ljflores !

**Review by ljflores — APPROVED** [review]

**Comment by ljflores** [comment]
LGTM

File: src/en/news/blog/2023/v17-2-6-quincy-released/index.md [hunk 2]
```diff
@@ -0,0 +1,1038 @@
...
+
+- ceph-volume: do not raise RuntimeError in util<span></span>.lsblk ([pr#50144](https://github.com/ceph/ceph/pull/50144), Guillaume Abrioux)
+
+- ceph-volume: fix a bug in get\_all\_devices\_vgs() ([pr#49453](https://github.com/ceph/ceph/pull/49453), Guillaume Abrioux)
+
+- ceph-volume: fix a bug in lsblk\_all() ([pr#49868](https://github.com/ceph/ceph/pull/49868), Guillaume Abrioux)
+
+- ceph-volume: legacy\_encrypted() shouldn't call lsblk() when device is 'tmpfs' ([pr#50161](https://github.com/ceph/ceph/pull/50161), Guillaume Abrioux)
+
+- ceph<span></span>.spec<span></span>.in: disable system\_pmdk on s390x for SUSE distros ([pr#48522](https://github.com/ceph/ceph/pull/48522), Tim Serong)
```
Comment (rzarzynski) [inline src/en/news/blog/2023/v17-2-6-quincy-released/index.md:52]: Why do we have the `<span></span>` tags here? Looks like HTML's junk. Are we trying to escape the dots?

File: src/en/news/blog/2023/v17-2-6-quincy-released/index.md [hunk 3]
```diff
@@ -0,0 +1,1038 @@
...
+---
+
+This is the sixth backport release in the Quincy series. We recommend all users update to this release.
+
+## Notable Changes
+
+- `ceph mgr dump` command now outputs `last_failure_osd_epoch` and
+  `active_clients` fields at the top level.  Previously, these fields were
+  output under `always_on_modules` field.
+
```
Comment (ljflores) [inline src/en/news/blog/2023/v17-2-6-quincy-released/index.md:17]: @yuriw sorry for the last addition, can you add this line under "Notable changes"?
```suggestion
telemetry: Added new metrics to the ‘basic’ channel to report  per-pool bluestore compression metrics. See a sample report with `ceph telemetry preview`. Opt-in with `ceph telemetry on`.
```

File: src/en/news/blog/2023/v17-2-6-quincy-released/index.md (hunk 2 above)
Comment (yuriw) [reply src/en/news/blog/2023/v17-2-6-quincy-released/index.md:56]: this is done via `ceph-release-notes` with `--markdown` option

**Pull request merged**